Tasks/CopyFilesOverSSHV0/copyfilesoverssh.ts

```diff
@@ -183,59 +217,86 @@ async function run() {
             }
         }
 
-        // identify the files to copy
-        const filesToCopy: string[] = getFilesToCopy(sourceFolder, contents);
-
-        // copy files to remote machine
-        if (filesToCopy) {
-            tl.debug('Number of files to copy = ' + filesToCopy.length);
-            tl.debug('filesToCopy = ' + filesToCopy);
-
-            let failureCount = 0;
-            console.log(tl.loc('CopyingFiles', filesToCopy.length));
-            for (const fileToCopy of filesToCopy) {
-                try {
-                    tl.debug('fileToCopy = ' + fileToCopy);
-
-                    let relativePath;
-                    if (flattenFolders) {
-                        relativePath = path.basename(fileToCopy);
-                    } else {
-                        relativePath = fileToCopy.substring(sourceFolder.length)
-                            .replace(/^\\/g, "")
-                            .replace(/^\//g, "");
-                    }
-                    tl.debug('relativePath = ' + relativePath);
-                    let targetPath = path.posix.join(targetFolder, relativePath);
+        if (contents.length === 1 && contents[0] === "**") {
```

Comment:
what does `contents[0] == "**"` mean?  sugget adding a comment

Reply:
This element is added when task's input **contents** has value '**'.
I've added a comment